gax-java/gax/src/main/java/com/google/api/gax/core/GaxProperties.java

```diff
@@ -144,7 +144,11 @@ static Optional<String> getBundleVersion(Class<?> clazz) {
         Attributes attributes = jar.getManifest().getMainAttributes();
         return Optional.ofNullable(attributes.getValue("Bundle-Version"));
       }
-    } catch (URISyntaxException | IOException e) {
+    } catch (URISyntaxException
+        | IOException
+        | IllegalArgumentException
+        | SecurityException
+        | NullPointerException e) {
```

Comment:
Can we just catch `Exception`? Is there any `Exception` that we would want to propagate from here?

Reply:
no, we should safely ignore this as it doesn't block runtime.